packages/mobile/views/dashboard/drawers/token/TokenRouter.svelte

```diff
@@ -0,0 +1,16 @@
+<script lang="typescript">
+    import { IAsset } from '@core/wallet'
+    import { onDestroy } from 'svelte'
+    import { tokenRoute, TokenRoute, tokenRouter } from '../../../../lib/routers'
+    import { TokenInformation } from './views'
+
+    export let asset: IAsset
+
+    onDestroy(() => {
+        $tokenRouter.reset()
```

Comment:
isnt this handled already in the dashboard router when the drawer closes/destroys?